SlevomatCodingStandard/Sniffs/Files/FileLineCountSniff.php

```diff
@@ -0,0 +1,62 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Files;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\FunctionHelper;
+use SlevomatCodingStandard\Helpers\SniffSettingsHelper;
+use function array_filter;
+use function array_keys;
+use function array_reduce;
+use function sprintf;
+use const T_OPEN_TAG;
+
+class FileLineCountSniff implements Sniff
+{
+
+	public const CODE_FILE_LINE_COUNT = 'FileLineCount';
+
+	/** @var int */
+	public $maxLinesLength = 250;
+
+	/** @var bool */
+	public $includeComments = false;
+
+	/** @var bool */
+	public $includeWhitespace = false;
+
+	/**
+	 * @return array<int, (int|string)>
+	 */
+	public function register(): array
+	{
+		return [T_OPEN_TAG];
+	}
+
+	/**
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.ParameterTypeHint.MissingNativeTypeHint
+	 * @param int $pointer
+	 */
+	public function process(File $phpcsFile, $pointer): void
+	{
+		$flags = array_keys(array_filter([
+			FunctionHelper::LINE_INCLUDE_COMMENT => $this->includeComments,
+			FunctionHelper::LINE_INCLUDE_WHITESPACE => $this->includeWhitespace,
+		]));
+		$flags = array_reduce($flags, static function ($carry, $flag): int {
+			return $carry | $flag;
+		}, 0);
+
+		$length = FunctionHelper::getLineCount($phpcsFile, $pointer, $flags);
+
+		if ($length <= SniffSettingsHelper::normalizeInteger($this->maxLinesLength)) {
```

Comment:
Please normalize the setting on the start of the `process` method, see https://github.com/slevomat/coding-standard/commit/0e4df34a76d26d3964c7654515b97a5b0f4cfb84